data/translations/es/state-data.json

```diff
@@ -522,17 +523,19 @@
     "registration_link": "https://www.elections.virginia.gov/citizen-portal/",
     "more_info_link": "https://www.elections.virginia.gov/es/registro/",
     "more_info_link_translated": "true",
-    "ip_deadline": "hasta 22 días antes del día de las elecciones",
-    "online_deadline": "hasta 22 días antes del día de las elecciones",
-    "mail_received_deadline": "a más tardar debe ser recibida 22 días antes del día de las elecciones",
-    "mail_postmarked_deadline": "",
+    "ip_deadline": "lunes, 17 de octubre de 2022",
+    "online_deadline": "lunes, 17 de octubre de 2022",
+    "mail_received_deadline": "",
```

Comment:
Updated from received to postmarked. Please check this. Also remove empty deadline properties.

Reply:
confirmed